internal/distro/rhel8/distro.go

```diff
@@ -691,12 +691,12 @@ func (t *imageType) checkOptions(customizations *blueprint.Customizations, optio
 	return nil
 }
 
-// New creates a new distro object, defining the supported architectures and image types
-func New() distro.Distro {
+// NewRHEL83 creates a new distro object, defining the supported architectures and image types
+func NewRHEL83() distro.Distro {
```

Comment:
On second look, this isn't right.  We've completely dropped 8.3.  The plain RHEL 8 should be doing the current GA.  This is all partially done in https://github.com/osbuild/osbuild-composer/pull/2830.  It's a bit stale now but I think we should figure out exactly what we're doing with these minor versions.

Reply:
My mistake, I missed the change that `rhel-8` and `rhel-9` now point to the latest GA release. I dropped those commits.